src/artificial_contrast/settings.py

```diff
@@ -0,0 +1,15 @@
+SEED = 42
+
+TEST_SET_EXAMINATIONS = [
```

Comment:
That shouldn't be part of repository IMO.
Maybe source file `test_set.csv` ?

Or maybe even better => test set patients should reside in a fully separated directory.
Then you just provide different data path
```
data_train
    P1
    P2
data_test
    P112
```